qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -23,6 +23,9 @@
 	// Error code for namespaces
 	UndefinedNamespace("No namespace resolver found for: `{0}`."), //
 
+	// Error code for operator
+	InvalidOperator("Invalid `{0}` operator for the section `#{1}`. Allowed operators are `{2}`."), //

```

Comment:
either `for section ``#{1}`` ` or `for the ``#{1}`` section `

Reply:
fixed